jetty-home/pom.xml

```diff
@@ -534,11 +507,6 @@
   </build>
   <dependencies>
     <!-- Orbit Deps -->
```

Comment:
At this point there aren't any more orbit deps, so we can get rid of the comment ;)